autogpt/workspace.py

```diff
@@ -39,7 +39,7 @@ def safe_path_join(base: Path, *paths: str | Path) -> Path:
     """
     joined_path = base.joinpath(*paths).resolve()
 
-    if CFG.restrict_to_workspace and not joined_path.is_relative_to(base):
+    if CFG.restrict_to_workspace and not joined_path.relative_to(base):
```

Comment:
`is_relative_to` can't be directly replaced by `relative_to`, as one returns `False` and the other throws a `ValueError` if the check fails. But this change also isn't necessary since we're on Python 3.10 now and don't officially support older versions.

Reply:
thanks, updated